index.html

```diff
@@ -10233,6 +10233,7 @@ <h5>Presentational Role Inheritance</h5>
               for details on implementing a tab set design pattern.
             </p>
             <p>Authors MUST ensure [=elements=] with <a>role</a> <rref>tab</rref> are <a>accessibility children</a> of an element with the role <rref>tablist</rref>.</p>
+            <p>Authors MUST ensure that if a <code>tab</code> is active, a corresponding <code>tabpanel</code> element that represents the active <code>tab</code> is rendered.</p>
```

Comment:
maybe this should also mention _not_ being in the collapsed state?

that's some multi-tab stuff that i'm not sure about though.... having an active tab but also having it be collapsed... is that common in this otherwise uncommon scenario?

Reply:
No one commented on my question here, so assuming it’s fine as is since it was merged